test_worm.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+import worm
+from common_types import Neighbors
+
+
+class WormTest(unittest.TestCase):
+    """General test class."""
+
+    def test_strike(self) -> None:
+        """Tests health change after strike."""
```

Comment:
Word **tests** here and on next lines is pretty bad because it sounds both like a noun and like a verb.

So It can be tests like "many test cases" and tests like "to check something". Prefer something different.